docs/building.md

```diff
@@ -84,6 +85,27 @@ sudo apt install cmake ninja-build python3 \
                  llvm
 ```
 
+## Prerequisites to build on macOS
+
+These steps were tested on macOS Catalina (10.15).
+
+First, you will need to install dependencies:
+
+- Install Python 3.x
```

Comment:
FWIW Catalina seems to include Python 3 already.
Since previous versions don't, I'm fine with keeping this line.